### PR TITLE
SJ INTERFACE TIDY UP

### PR DESCRIPTION
**Pull request by theomarkkuspaul**
What: Sort order 'latest at the top' for snippets
Why:  SJ Interface tidy up

This change addresses the need by:
* Add #latest scope method to Snippet model
* Add spec for latest method
* Disable automatic ordering to Snippet datatable
* Use #latest method in Snippet index action

### CODE REVIEW

File: app/assets/stylesheets/parsers.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 #preview-modal {
   padding-top: 16px;
+  height: 600px;
```
Comment (yarhtut) [inline app/assets/stylesheets/parsers.scss]: `rem`?

File: app/models/collection_statistics.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@ def self.index_statistics(statistics)
 			index_statistics[stats_item.day][:deleted] += stats_item.deleted_count
 		end
 
-		index_statistics
+    index_statistics
```
Comment (yarhtut) [inline app/models/collection_statistics.rb]: indentation of that method looks off.

File: app/models/concerns/orderable.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+# app/models/concerns/orderable.rb
+module Orderable
+  extend ActiveSupport::Concern
+
+  included do
+    scope :latest, -> { order(updated_at: 'desc') }
+  end
+
+end
```
Comment (yarhtut) [inline app/models/concerns/orderable.rb]: new line

File: app/assets/stylesheets/previews.scss [hunk 4]
```diff
@@ -15,7 +15,7 @@
 }
 
 .tabs-panel {
-  max-height: 50vh;
+  max-height: 400px;
```
Comment (yarhtut) [inline app/assets/stylesheets/previews.scss]: `px` .?

File: app/models/collection_statistics.rb (hunk 2 above)
Comment (theomarkkuspaul) [reply app/models/collection_statistics.rb]: 👍

**Comment by motizuki** [comment]
👍 @theomarkkuspaul good work!

**Pull request merged**